backend/routes/canvas.go

```diff
@@ -30,23 +40,32 @@ func initCanvas(w http.ResponseWriter, r *http.Request) {
 		// Create canvas
 		canvas := make([]byte, totalByteSize)
 		ctx := context.Background()
-		err := core.ArtPeaceBackend.Databases.Redis.Set(ctx, "canvas", canvas, 0).Err()
+		err := core.ArtPeaceBackend.Databases.Redis.Set(ctx, canvasKey, canvas, 0).Err()
 		if err != nil {
 			routeutils.WriteErrorJson(w, http.StatusInternalServerError, "Failed to initialize canvas")
 			return
 		}
 
-		routeutils.WriteResultJson(w, "Canvas initialized")
+		routeutils.WriteResultJson(w, fmt.Sprintf("Canvas for round %s initialized", roundNumber))
 	} else {
-		routeutils.WriteErrorJson(w, http.StatusConflict, "Canvas already initialized")
+		routeutils.WriteErrorJson(w, http.StatusConflict, fmt.Sprintf("Canvas for round %s already initialized", roundNumber))
 	}
 }
 
 func getCanvas(w http.ResponseWriter, r *http.Request) {
 	routeutils.SetupAccessHeaders(w)
 
+	// Get round number from query params, default to latest round if not specified
+	roundNumber := r.URL.Query().Get("round")
+	if roundNumber == "" {
+		// TODO: Implement logic to get latest round number
+		roundNumber = "1" // Default to round 1 for now
```

Comment:
We an also default to the current round set here : `core.ArtPeaceBackend.CanvasConfig.Round`

Reply:
Okay boss resolving these now